app/views/compute_resources/form/_ec2.html.erb

```diff
@@ -3,6 +3,8 @@
 <%= text_f f, :user, :label => _("Access Key"), :help_inline => documentation_button('5.2.3EC2Notes') %>
 <%= password_f f, :password, :label => _("Secret Key"), :unset => unset_password? %>
 
+<%= checkbox_f f, :gov_cloud, :label_help => 'AWS GovCloud (US) is an isolated AWS region designed for U.S. government agencies<br>addressing their specific regulatory and compliance requirements.<br><br>If you want to use GovCloud region, check this. For all other regions, keep this unchecked.' %>
```

Comment:
This essentially makes the region field redundant, correct? Might be nice to disable it when checked.

Reply:
If there is a possibility there could be more regions (it's just an implementation detail there is just one today), I'm ok with keeping this as it is.

Reply:
Well, I have no indication they are adding one in future, though today with non gov-cloud we in fact ask the default region us-west to give us list of all available regions including that us-west. In this case we ask gov-cloud give us the list returning only gov-cloud. In this scenario, it's more like a "test connection" functionality that verifies credentials. So it might make sense to show green test connection instead of grey load regions that after first call changes to green test connection...

Since this is a one-time setup, I'd prefer to not over complicate this. The JS that does the test connection magic is already complicated enough.